quarkus/container/README.md

```diff
@@ -0,0 +1,71 @@
+# Keycloak Image
+
+The `Dockerfile` for the Keycloak container image.
+
+See the `Build` section for more details on how to build the image.
+
+Once built, you can run the server in the same manner as when using the Keycloak distribution by passing any command-line argument:
+
+    docker run --name keycloak -p 8080:8080 \
+           -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=change_me \
+           IMAGE[:TAG] \
+           start-dev
+
+## Extending the Image
+
+To customize the base image, create a new `Dockerfile` similar to following:
+
+```
+FROM quay.io/keycloak/keycloak
+
+WORKDIR /opt/keycloak
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+```
+
+In the example above, you are basically creating a new image `FROM` this image and adding a Java Keystore to configure HTTPS using a self-signed certificate (only for example purposes, never do that for production).
+
+As another example, you can install any custom provider you may have (including themes within a JAR file) as follows:
+
+```
+FROM quay.io/keycloak/keycloak
+
+COPY my-providers/ /opt/keycloak/providers/
+
+WORKDIR /opt/keycloak
+
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+
+# Run the build command to install custom providers
+RUN ./bin/kc.sh build
+```
+
+In the example above, any JAR files within the directory `my-providers` will be copied to the image's `/opt/keycloak/providers` directory. Then the `build` command is executed to install the custom providers.
+
+You can configure the server using any configuration option available in the CLI as follows:
+
+```
+FROM quay.io/keycloak/keycloak
+
+COPY my-providers/ /opt/keycloak/providers/
+
+WORKDIR /opt/keycloak
+
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+
+RUN ./bin/kc.sh build --db=postgres && ./bin/kc.sh start-dev --db-url=jdbc:postgresql://$DB_HOST/keycloak --db-username=keycloak --db-password=password
+```
+NOTE: Note that we are using separated `RUN` steps for illustrative purposes, ideally you should reduce the number of layers and execute all steps in a single one.
+
+In the example above, Keycloak is set to use e a PostgreSQL database in development mode. You can for sure also use environment variables for the configuration and build an optimized image for better startup time. For more information, see the [Running Keycloak in a container guide](https://www.keycloak.org/guides/server/containers).
+
+
+## Build the image
```

Comment:
```suggestion

```

I'd remove everything around running and extending the image, as that should be covered by the guides. This should just be about building it.

Reply:
done